package.json

```diff
@@ -13,6 +13,7 @@
 		"clean:logs": "rimraf ./Logs",
 		"clean:dist": "rimraf ./dist",
 		"clean:coverage": "rimraf ./coverage",
+		"clean:all": "rimraf ./build ./Logs ./dist ./coverage",
```

Comment:
Can you use `concurrently` and call the other scripts instead of calling `rimraf` directly?

Reply:
Sure, okay.